### PR TITLE
Readme/update hyperledger

### PR DESCRIPTION
**Pull request by Patrik-Stas**

### CODE REVIEW

**Review by ryjones — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -18,15 +20,15 @@
 
 # Get started
 The best way to get your hands on.  
-* NodeJS [demo](https://github.com/AbsaOSS/libvcx/tree/master/wrappers/node) - NodeJS wrapper is our first class citizien 
+* NodeJS [demo](https://github.com/hyperledger/aries-framework-rs/tree/master/wrappers/node) - NodeJS wrapper is our first class citizien
```
Comment (mirgee) [inline README.md:23]: I think we should choose more specific and accurate formulations in the docs - something like 'NodeJS wrapper is currently the most up to date wrapper', or 'NodeJS wrapper is currently our first priority'. Alternatively, leave this out completely and only mention that the Python wrapper has been unmaintained.  Secondly, `citizien` sounds like a failed attempt at French :)

File: README.md [hunk 2]
```diff
@@ -18,15 +20,15 @@
...
+* NodeJS [demo](https://github.com/hyperledger/aries-framework-rs/tree/master/wrappers/node) - NodeJS wrapper is our first class citizien 
 * Android [demo](https://github.com/sktston/vcx-demo-android)  (3rd party demo)
 * iOS [demo](https://github.com/sktston/vcx-demo-ios) (3rd party demo)
 * iOS [skeleton project](https://github.com/sktston/vcx-skeleton-ios) (3rd party demo)
 
 #### ::Important::
 However before you'll be able to pick one of these demos and run them locally, you need to build binary library which
 all these demos depends on.  
--  [Building AriesFrameworkRs on OSX, Linux](./docs/build-general.md)
+- [Building AriesFrameworkRs on OSX, Linux](./docs/build-general.md)
```
Comment (mirgee) [inline README.md:31]: Be consistent with 'casing'

File: README.md [hunk 3]
```diff
@@ -37,6 +39,6 @@ all these demos depends on.
 - The artifacts are produced from every PR (unless it's coming from a forked repository due to limitations of Github Actions).
 
 - NodeJS wrappers are published on [npmjs](https://www.npmjs.com/package/@hyperledger/node-vcx-wrapper)
-- Docker images are in [Github Packages](https://github.com/AbsaOSS/libvcx/packages/332720/versions)
-- Mobile artifacts are attached to [CI runs](https://github.com/AbsaOSS/libvcx/actions) (click on particular CI run to
+- Docker images are in [Github Packages](https://github.com/hyperledger/aries-framework-rs/packages)
+- Mobile artifacts are attached to [CI runs](https://github.com/hyperledger/aries-framework-rs/actions) (click on particular CI run to
```
Comment (mirgee) [inline README.md:43]: Maybe also mentiod that they are part of the release assets

File: README.md [hunk 4]
```diff
@@ -1,15 +1,17 @@
-# Aries Framework Rust
+# <img alt="Hyperledger Aries logo" src="docs/aries-logo.png" width="45px" /> Aries Framework Rust
 
-![CI build](https://github.com/AbsaOSS/libvcx/workflows/CI/badge.svg)
-[![codecov](https://codecov.io/gh/AbsaOSS/libvcx/branch/master/graph/badge.svg)](https://codecov.io/gh/AbsaOSS/libvcx)
+![CI build](https://github.com/hyperledger/aries-framework-rs/workflows/CI/badge.svg)
+[![codecov](https://codecov.io/gh/hyperledger/aries-framework-rs/branch/master/graph/badge.svg)](https://codecov.io/gh/hyperledger/aries-framework-rs)
 
-- Aries c-callable implementation written in Rust with language wrappers available in:
+Join our [Hyperledger RocketChat channel](https://chat.hyperledger.org/channel/aries-framework-rust)
```
Comment (mirgee) [inline README.md:6]: We are quite inconsistent in naming of the library. I can easily someone being confused if there is a difference between "aries-framework-rust" and "aries-framework-rs"

**Review by mirgee — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Patrik-Stas) [reply README.md:6]: @ryjones  is it possible to change the name of rocketchat channel `aries-framework-rust` to `aries-framework-rs` for consistency?

Comment (ryjones) [reply README.md:6]: Yes

**Pull request merged**